shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/Server.java

```diff
@@ -486,6 +529,42 @@ private void addJson(String url, Function<ObjectMapper, JsonNode> fetcher) {
 		return functionpRepository.stream();
 	}
 
+	private String localname() {
+		URIBuilder builder = null;
+		final String url = System.getenv("LOCAL_URL");
```

Comment:
`LOCAL_URL` doesn't seem to exist anywhere else in the Shesmu code base; should it be described in the installation docs?

Reply:
Added to the README.